conda_forge_tick/make_graph.py

```diff
@@ -324,6 +325,44 @@ def make_graph(names: List[str], gx: Optional[nx.DiGraph] = None) -> nx.DiGraph:
     return gx
 
 
+def _get_last_updated_prs():
+    query = textwrap.dedent("""
+        {
+          user(login: "regro-cf-autotick-bot") {
+            pullRequests(first: 100, states: [CLOSED, MERGED], orderBy: {field: UPDATED_AT, direction: DESC} ) {
```

Comment:
Would it be possible to have a query that gets PRs with the bot-rerun label?

Reply:
Yes, you can filter by label.